lib/acx/enforcer_server.ex

```diff
@@ -340,12 +340,14 @@ defmodule Acx.EnforcerServer do
   # Returns a tuple used to register and lookup an enforcer process
   # by name
   defp via_tuple(ename) do
-    {:via, Registry, {Acx.EnforcerRegistry, ename}}
+    {type, name} = Application.get_env(:acx, :registry, {Registry, Acx.EnforcerRegistry})
+    {:via, type, {name, ename}}
```

Comment:
@cpelletier-cogility so does this mean that my other `horde` PR for this repo could be closed. Because everything else would be inside the `sub-portal-otp` PR ?

Reply:
Yes. I don't know how much I like this approach but for time's sake it should work for now. It seems like most libraries use watchers to handle things like this. We have a postgres one in our node app but after messing with it the other day it doesn't seem to be working as intended. It may be work developing an elixir version though when we have time. Either that or something that uses redis pubsubs.